pydis_site/hosts.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django_hosts import host, patterns
+
+host_patterns = patterns(
+    '',
+    # > | Subdomain | URL Module   | Host entry name |
```

Comment:
this kind of comment is sort of an unusual thing to have.

wait, did you create this file for this PR? That's what the changes makes it look like, but that seems odd since it's not really part of any of those issues?

Reply:
I did not create this file, no.